### DISCUSSION TITLE
Copilot payment for $10 was unexpectedly refunded

### DISCUSSION THREAD

**Discussion by jamiolee** [community/community#30743] [New to GitHub] [answered]
I have no idea how to contact someone at GitHub, their email just pointed to here. I was on the Copilot invite for a while then recently had to cough up the $10/month, I did so happily and now I just got an email from GitHub saying that my $10 has been refunded with no explanation as to why. How do I contact someone to figure out what is going on here?

**Comment by thomas-ayissi** [comment, marked as answer]
Hello,
At least two things can help you understand what is going on:

**Option 1/** Look in the mailbox linked to your account for the GitHub email from the time the $10 was returned to your account.
Just type "Github" in the search bar above your mail and you will certainly find an explanatory mail. If you read it, you will understand what happened.

**Option 2 /** Click on your profile icon at the top right of the homepage. Select "settings" and then click on "Billing and Plans" in the sidebar on the left of your screen.
At the top of the page that will open, you will see the "Personal Billing" section where you can access the details of your financial transactions. By checking the current nature of your subscription and if necessary downloading the available documents, you will understand what has happened.

Have a nice weekend

**Reply by jamiolee** [reply]
Thank you, turns out they double charged my card, I didn't notice that they sent me two email receipts, one immediately after the other.

**Reply by Abdiisaba057** [reply]
Thank you

**Comment by emploiyer** [comment]
ok

**Reply by nhoj05** [reply]
Pls

**Comment by emploiyer** [comment]
I will send message aon condination to verify ok understood your


Le ven. 26 août 2022 à 00:44, Thomas Ayissi ***@***.***> a
écrit :

**Comment by john9384** [comment]
Hello does anyone know the GitHub customer care email?
My account was billed three times but yet my GitHub copilot is not working. $10 dollars each. I got a mail saying there was a problem with my payment but I was deducted still. How do I reach out to lay complaints? I followed the contact link in the mail but that brought me here.
Any help would be highly appreciated pls.
